WcaOnRails/config/locales/de.yml

```diff
@@ -2387,6 +2458,10 @@ de:
       name_too_long: >-
         Der Name des Wettbewerbs ist länger als 32 Zeichen. Wir bevorzugen kurze
         Namen und würden uns sehr freuen, wenn Du ihn anpasst.
+      #original_hash: af27396
+      id_starts_with_lowercase: >-
+        Die Competition-Id beginnt mit einem Kleinbuchstaben. Wir bevorzugen den
+        Beginn mit einem Großbuchstaben.
```

Comment:
Did not want to write am Wortanfang originally, because that could mean (in the inclusive sense): "for all words, please start with an uppercase letter" although here, it's only pointing to the start of the comp. id. Will use your first suggestion nonetheless in the next update.